cmsmon-tools.spec

```diff
@@ -0,0 +1,76 @@
+### RPM cms cmsmon-tools 0.3.10
+## INITENV +PATH PYTHONPATH %i/${PYTHON_LIB_SITE_PACKAGES}
+## NOCOMPILER
+
+%define pkg CMSMonitoring
+%define ver %realversion
+%define cmsmon_commands nats-sub nats-pub nats-exitcodes-termui dbs_vm
+%define flags -ldflags="-s -w -extldflags -static"
+Source0: https://github.com/dmwm/%pkg/archive/%ver.tar.gz
+
+Requires: go
+
+# RPM macros documentation
+# http://www.rpm.org/max-rpm/s1-rpm-inside-macros.html
+%prep
+%setup -D -T -b 0 -n %pkg-%ver
+
+%build
+export GOCACHE=%{_builddir}/gocache
+cd ..
+cd %pkg-%ver
+echo "build $PWD"
+ls
+mkdir -p gopath/bin
+export GOPATH=$PWD/gopath
+go get github.com/dmwm/cmsauth
+go get github.com/vkuznet/x509proxy
+go get github.com/sirupsen/logrus
+go get github.com/prometheus/client_golang/prometheus
+go get github.com/prometheus/common/log
+go get github.com/prometheus/common/version
+go get github.com/shirou/gopsutil/cpu
+go get github.com/shirou/gopsutil/mem
+go get github.com/shirou/gopsutil/load
+go get github.com/shirou/gopsutil/process
+go get github.com/go-stomp/stomp
+go get github.com/nats-io/nats.go
+go get github.com/gizak/termui/v3
+
+# build monit tools
+cd src/go/MONIT
+go build %flags monit.go
```

Comment:
you are building monit but not copying it in the distribution,

Reply:
it is copied here:
https://github.com/cms-sw/cmsdist/pull/5750/files#diff-c13d4ca5c238805a5e42ba8af94d9b5bR55

Reply:
this is copying in %i. We need to create a wapper for this in %post too. Anyway, I will fix it.